iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/RatioTapChangerImpl.java

```diff
@@ -97,27 +101,63 @@ public RatioTapChangerImpl setLoadTapChangingCapabilities(boolean loadTapChangin
 
     @Override
     public double getTargetV() {
-        return targetV.get(network.get().getVariantIndex());
+        if (regulationMode != RegulationMode.VOLTAGE) {
+            throw new IllegalAccessError("Regulation mode must be in voltage to access to target V");
```

Comment:
Use a constant for message.

Reply:
message not needed anymore, changed to return Double.NaN if regulation mode is not voltage